pkg/builder/local_build_executor.go

```diff
@@ -248,6 +253,24 @@ func (be *localBuildExecutor) Execute(ctx context.Context, filePool re_filesyste
 		return response
 	}
 
+	if len(be.inputRootCharacterDevices) > 0 {
+		if err := inputRootDirectory.Mkdir("dev", 0777); err != nil && !os.IsExist(err) {
+			attachErrorToExecuteResponse(response, util.StatusWrap(err, "Unable to create dev directory in input root"))
```

Comment:
Any error prior to execution should lead to direct termination. Don't forget to `return response` here!